src/shared/layout/header/header.scss

```diff
@@ -2,14 +2,14 @@ $header-color: #fff;
 $header-color-secondary: #bbb;
 $header-color-hover: darken($header-color, 20%);
 
-.hud-logo {
+.logo-back {
   display: inline-block;
   width: 50%;
   height: 500px;
   margin-left: 22rem;
   margin-bottom: 0;
   padding-bottom: 0;
-  background: url('../../../content/images/hud-logo.png') no-repeat center top;
+  background: url('../../../content/images/HUD-logo.png') no-repeat center top;
```

Comment:
can you commit HUD-logo.png?  Or change the case of the logo name.  
Failed to compile
./src/shared/layout/header/header.tsx
Cannot find file: 'HUD-logo.png' does not match the corresponding name on disk: '.\src\content\images\hud-logo.png'.